contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -749,4 +757,13 @@ contract GenesisProtocolLogic is IntVoteInterface {
                 (pState == ProposalState.Queued)
         );
     }
+
+    /**
+      * @dev calcExecuteCallBounty calculate the execute boosted call bounty
+      * @param _proposalId the ID of the proposal
+      * @return uint256 executeCallBounty
+    */
+    function calcExecuteCallBounty(bytes32 _proposalId) private view returns(uint256) {
```

Comment:
If there's no special reason not to, please make this `public`.

Reply:
ok. maybe it will be useful to someone.